script.sh

```diff
@@ -29,10 +29,16 @@ echo '::group::Preparing ...'
 
   TEMP_PATH="$(mktemp -d)"
   echo "Detected ${os} running on ${arch}, will install tools in ${TEMP_PATH}"
+  JQ_PATH="${TEMP_PATH}/jq"
   REVIEWDOG_PATH="${TEMP_PATH}/reviewdog"
   TFSEC_PATH="${TEMP_PATH}/tfsec"
 echo '::endgroup::'
 
+echo "::group:: Installing jq ... https://stedolan.github.io/jq"
```

Comment:
We do not need to incall `jq` as every Available Environment has jq by default. https://github.com/actions/virtual-environments#available-environments

Reply:
is fixed can you re-run pipeline ? :)

Reply:
oh, i see. 

It's ok to include installing jq script when it's not available and it support windows/mac.